test/Inputs/clang-importer-sdk/usr/include/dispatch.h

```diff
@@ -1,30 +0,0 @@
-#import <objc/NSObject.h>
```

Comment:
This seems suspect to me. test/Inputs/clang-importer-sdk is a fake SDK under the control of the tests, which means anything depending on this should *not* be using the real Dispatch.